source/particle/integrator/rk_4.cc

```diff
@@ -197,7 +197,8 @@ namespace aspect
       ASPECT_REGISTER_PARTICLE_INTEGRATOR(RK4,
                                           "rk4",
                                           "Runge Kutta fourth order integrator, where "
-                                          "$y_{n+1} = y_n + (1/6)*k_1 + (1/3)*k_2 + (1/3)*k_3 + (1/6)*k_4$ "
+                                          "$y_{n+1} = y_n + \\frac{1}{6} k_1 + \\frac{1}{3} k_2 "
+                                          "           + \\frac{1}{3} k_3 + \\frac{1}{6} k_4$ "
```

Comment:
why do you start with this much whitespace? This will look weird without the newline, doesn't it?

Reply:
Latex don't care about spaces in formulas. I just wanted it to be readable in the source files :-)

Reply:
My point is that it looks like this in the prm and gui now:
```
    # `rk4': Runge Kutta fourth order integrator, where $y_{n+1} = y_n +
    # \frac{1}{6} k_1 + \frac{1}{3} k_2            + \frac{1}{3} k_3 +
    # \frac{1}{6} k_4$ and $k_1$, $k_2$, $k_3$, $k_4$ are defined as usual.
```

Reply:
Ah, hm. That's unfortunate. We really ought to come up with a way to print real formulas in the GUI -- Qt can do everything, can't it do that as well?

Either way, I've fixed this as well.